Circumference.java

```diff
@@ -1,13 +1,24 @@
 class Circumference extends ConsoleProgram {
 
   /**
-  * Description
-  * @author:
+  * A program that computes the curumference of a circle given a radius from the user 
```

Comment:
Change description of code to: "This program computes the circumference of a circle when given a radius."